packages/jaeger-ui/src/components/TracePage/TraceStatistics/tableValues.test.js

```diff
@@ -506,12 +506,12 @@ describe('check self time', () => {
       'Service Name',
       transformedTraceWithOverlappingChildrenLongerThanParent
     );
-    expect(serviceOne.selfTotal).toBe(10.33);
+    expect(serviceOne.selfTotal).toBe(0.03);
```

Comment:
If all the timestamps are integers, how do we end up with these fractional values?

Reply:
https://github.com/jaegertracing/jaeger-ui/blob/1f63ee05ac987487d9207f7dba4626f50cceadf7/packages/jaeger-ui/src/components/TracePage/TraceStatistics/tableValues.tsx#L245-L260

becuase column values are calculated from this function finally.

Reply:
Do you mean that this is a value divided by 1000? If so that's testing the display behavior, but we should have unit tests that validate calculations themselves, in the original units (which would be integers)

Reply:
Yes but every test in tableValues.test.js is testing display behaviour i.e, each column value we see in TraceStatistics view
https://github.com/jaegertracing/jaeger-ui/blob/1f63ee05ac987487d9207f7dba4626f50cceadf7/packages/jaeger-ui/src/components/TracePage/TraceStatistics/tableValues.test.js#L80-L100